server/routerlicious/packages/lambdas/src/alfred/interfaces.ts

```diff
@@ -0,0 +1,70 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { IEvent } from "@fluidframework/common-definitions";
+import { IClient, IConnected } from "@fluidframework/protocol-definitions";
+
+/**
+ * Connection details of a client
+ */
+export interface IConnectedClient {
+	/**
+	 *
+	 */
+	connection: IConnected;
```

Comment:
@znewton  @hedasilv @pradeepvairamani do you have some context to add here apart from the inherited docs of IConnected?

The inherited docs say the following for IConnected: 
```
/**
 * Message sent to indicate a client has connected to the server
 */
```

Reply:
Following the example of the inherited `IClient` docs, we could add a bit more to briefly summarize what's included.
```
/**
 * Message sent to indicate a client has successfully connected to the server.
 * Includes client and service connection details to establish communication limits and expectations. 
 */
```